Cron node added in env.php reference (#7109)
magento/devdocs: src/guides/v2.3/config-guide/prod/config-reference-envphp.md

Learn more about different [Cache Types][cache-types].

## cron

> This node is used to enable/disable the cron jobs. `0` means disabled and `1` means enabled. If disabled, jobs will not run.

Reviewer: I would also mention the default value if this config does not exist

> This node is used to enable/disable the cron jobs. `0` means disabled and `1` means enabled. If disabled, jobs will not run.

Reviewer 2: ```suggestion
Enable or disable cron jobs for the Magento application. By default, cron jobs are enabled. To disable them, add the `cron` configuration to the `env.php` configuration and set the value to `0`. 

{:.bs-callout-warning}
Disabling cron jobs means essential processes required by the Magento application will not run. To disable crons temporarily, you can use the Magento `cron:remove` Magento CLI command instead of disabling them in the `env.php` file. See [Configure and run cron]/{{site.baseurl}}/config-guide/cli/config-cli-subcommands-cron.html).
```

Reviewer 3: Out of curiosity: what is the difference between disabling cronjobs using `env.php` and by executing `cron:remove`. Can this be clarified somehow?

Author: @hostep : I think both are similar. Same like, we can change status of the cache type and Magento mode using cli command and using env.php as well.
as @meker12 said, it's not recommended to make setting changes in env.php file because it is not feasible way. But that is also one of the way.

Reviewer 2: I think it's just running command from command line rather than changing the env.php-- but you also have to run cron:install to restore them.  It's explains how to do it in the link. Maybe instead of that note with detail, it might be better to just say that you can also temporarily disable and enable crons using Magento CLI commands -- and include the link.

Reviewer 3: Be aware `cron:install` and disabling them in the `env.php` files are not the same.

`cron:install` will tell the operating system of the server that it should add certain entries to the crontab of the system. Then the operating system will automatically call `bin/magento cron:run` every minute.
Calling `cron:remove` will remove them again from the crontab. And no further calls to `bin/magento cron:run` will happen.

However, if you disable cronjobs in the `env.php` file, the operating system will still call `bin/magento cron:run`, but Magento will ignore that call and not perform any actions because it first checks that flag in the `env.php` file (that's what the code does linked in the opening post).

I'm not aware of a command line command to manipulate the cron enabled flag in the `env.php` file though, we just parse the file, manipulate it and save it again in an automated fashion.

Author: Great info @hostep. Thanks.

> This node is used to enable/disable the crons. `0` means disable and `1` means enable. If crons are disabled, Jobs will not run.

Reviewer 4: I'm not sure "crons" is a correct definition of the multiple jobs. Please, use "cron jobs" instead of "crons" since "crons" literally means multiple schedulers.

> This node is used to enable/disable the crons. `0` means disable and `1` means enable. If crons are disabled, Jobs will not run.

Reviewer 4: ```suggestion
This node is used to enable/disable the crons. `0` means disabled and `1` means enabled. If crons are disabled, jobs will not run.
```

```conf
'cron' => [
  'enabled' => 0
]
```

> 
> Learn more about [Crons][crons].
> 

Reviewer 2: Added the reference to the warning note in previous suggestion, so you can remove it here.

```suggestion

```
